### ISSUE TITLE
trinity_kmer_length config option not working

### ISSUE THREAD

**Issue by HumzaHemani** [Teichlab/tracer#102]
Hi,

Setting the trinity_kmer_length option in the config file leads to this error from trinity: 
`ERROR, don't recognize parameter: --KMER_SIZE`
Based on [this](https://groups.google.com/g/trinityrnaseq-users/c/XUxfT96zpds/m/a7QFeKbJBQAJ), I believe that `--KMER_SIZE` should be changed to `--__KMER_SIZE` on line 1093 of tracer_func.py. 
When running Trinity standalone instead of Tracer, using --__KMER_SIZE instead of --KMER_SIZE works.

Thank you!

**Comment by mstubb** [comment]
Yes, this has obviously been changed in an update to Trinity. You can either:

1. Install the last version of Trinity that accepted the old argument

or

2. Edit the tracer code so that it passes the new argument to the current version of Trinity. 

Cheers,

Mike
